package.js

```diff
@@ -694,44 +678,40 @@ class PathToTarkovController {
-                        }
-                        if (locations[i].base.exits[x].RequirementTip !== '') {
-                            locations[i].base.exits[x].RequirementTip = '';
-                        }
-                        if (locations[i].base.exits[x].RequiredSlot) {
-                            delete locations[i].base.exits[x].RequiredSlot;
-                        }
-                    }
-
-                    // Make all extractions available to extract
-                    if (locations[i].base.exits[x].Name !== "EXFIL_Train") {
+                    if (locations[i].base.exits[x].Name !== "EXFIL_Train" && !locations[i].base.exits[x].Name.includes("lab") || locations[i].base.exits[x].Name === "lab_Vent") {
                         if (locations[i].base.exits[x].Chance !== 100) {
                             locations[i].base.exits[x].Chance = 100;
                         }
+                        if (this.config.remove_all_exfils_restrictions) {
+                            if (locations[i].base.exits[x].ExfiltrationTime !== 10) {
+                                locations[i].base.exits[x].ExfiltrationTime = 10;
+                            }
+                            if (locations[i].base.exits[x].PlayersCount !== 0) {
+                                locations[i].base.exits[x].PlayersCount = 0;
+                            }
+                            if (locations[i].base.exits[x].Id !== '') {
+                                locations[i].base.exits[x].Id = '';
+                            }
+                            if (locations[i].base.exits[x].Count !== 0) {
+                                locations[i].base.exits[x].Count = 0;
+                            }
+                            if (locations[i].base.exits[x].PassageRequirement !== "None") {
+                                locations[i].base.exits[x].PassageRequirement = "None";
+                            }
+                            if (locations[i].base.exits[x].ExfiltrationType !== "Individual") {
+                                locations[i].base.exits[x].ExfiltrationType = "Individual";
+                            }
+                            if (locations[i].base.exits[x].RequirementTip !== '') {
+                                locations[i].base.exits[x].RequirementTip = '';
+                            }
+                            if (locations[i].base.exits[x].RequiredSlot) {
+                               delete locations[i].base.exits[x].RequiredSlot;
+                            }
+                        }
                     }
                 }
             }
         }
-
-        const database = DatabaseServer.tables;
-
-        Object.keys(this.config.exfiltrations).forEach(mapName => {
-            const extractPoints = Object.keys(this.config.exfiltrations[mapName]);
-            database.locations[mapName].base.exits = extractPoints.map(createExitPoint(this.entrypoints[mapName]));
-        });

```

Comment:
Do not remove this

Reply:
I have with no ill reactions to the server or spawning bots or loading into maps?

Reply:
The problem is here, just keep this lines behind a `if (this.config.remove_all_exfils_restrictions)` should do the job

Reply:
> I have with no ill reactions to the server or spawning bots or loading into maps?

Not sure to understand your message, but you just removed a feature right here.

Reply:
if your talking about the all extracts to be open the code already does that.